### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by nicosomb**

### CODE REVIEW

**Comment by j0k3r** [comment]
Did you cover all the missing translation?

**Comment by nicosomb** [comment]
No, this PR is in WIP (I forgot WIP in the title, sorry)

File: src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig [hunk 1]
```diff
@@ -76,40 +77,3 @@
...
-            </div>
-        </div>
-        <div class="footer-copyright">
-            <div class="container">
-                <p>{% trans %}powered by{% endtrans %} <a target="_blank" href="https://wallabag.org" class="grey-text text-lighten-4">wallabag</a></p>
-                <a class="grey-text text-lighten-4 right" href="{{ path('about') }}">{% trans %}about{% endtrans %}</a>
-            </div>
-        </div>
-    </footer>
-{% endblock %}
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig]: Why did you remove the footer ?

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig]: I think we'll remove or change its content, but you're right, nothing to do here, I made a mistake.

Comment (nicosomb) [inline src/Wallabag/CoreBundle/Resources/views/themes/material/layout.html.twig]: restored
